app/models/package_manager/maven.rb

```diff
@@ -7,6 +7,9 @@ class Maven < Base
     URL = "http://maven.org"
     BASE_URL = "https://maven-repository.com"
     COLOR = '#b07219'
+    LICENSE_STRINGS = {
+      "http://www.apache.org/licenses/LICENSE-2.0" => "Apache-2.0"
```

Comment:
Can you add the Eclipse license in here too? "http://www.eclipse.org/legal/epl-v10.html"